websites/M/MangaLife/presence.ts

```diff
@@ -0,0 +1,150 @@
+const presence = new Presence({ clientId: "906057236927893576" }),
+  browsingStamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+  const [privacy, cover, timestamps, buttons] = await Promise.all([
+    presence.getSetting<boolean>("privacy"),
+    presence.getSetting<boolean>("cover"),
+    presence.getSetting<boolean>("timestamps"),
+    presence.getSetting<boolean>("buttons")
+  ]);
+  let presenceData: PresenceData = {
+    largeImageKey: "logo",
+    smallImageKey: !privacy ? "home" : "",
+    details: "Browsing...",
+    state: "at Home Page",
+    startTimestamp: browsingStamp
+  };
+  const path = document.location.pathname,
+    pages: Record<string, PresenceData> = {
+      "/manga/": {
+        details: "Viewing a Manga...",
+        state: document.title.split(" | MangaLife")[0],
+        startTimestamp: browsingStamp
```

Comment:
This is set in the `presenceData` object on L11, you do not need to redefine this. Remove all matching instances too

Reply:
also didn't notice, it was pretty late and i may have rushed a lot. will also fix asap.